src/_css/PageNotFoundComponent.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css?family=Mitr&effect=destruction');
+@import url('https://fonts.googleapis.com/css?family=Frijole');
+
+.page-404 {
+  background: linear-gradient(-20deg, #4e4376 0%, #3359df 100%);
+  height: 100vh;
+}
+
+.button-404 {
+  margin: 14px !important;
+}
+
+.card-404 {
+  top: 30%;
+  left: 0%;
+  transform: scale(2,2);
+  width: 20% !important;
+  height: 32%;
+}
+
+.ui.card .meta {
+  padding-top: 0px;
+  font-family: 'Mitr', sans-serif;
+  font-size: 0.5vw;
+  font-weight: normal;
+}
+
+.ui.card>.content, .ui.cards>.card>.content {
+  padding-top: 30px;
+}
+
+.ui.card>.content p, .ui.cards>.card>.content p {
+  margin: 0px;
+  font-family: 'Frijole', cursive;
+  font-size: 5.6vw;
+  color: #fff;
+  /* color: black; */
+}
+
+.ui.card {
+  box-shadow: 2px 2px 81px -8px rgba(0,0,0,0.65) !important;
+  background: none;
+  border-radius: 0px;
+}
+
+.ui.card>.content>.description,
+ .ui.cards>.card>.content>.description {
+   padding-top: 18px;
+   font-family: 'Mitr', sans-serif;
+   font-size: 0.6vw;
+   font-weight: normal !important;
+   color: rgba(0,0,0,.4);
+ }
+
+#andela-logo-404 {
+    position: absolute;
```

Comment:
Indentation for `#andela-logo-404` is off. It's supposed to be `2 spaces for each level of indentation` not `4 spaces`